### DISCUSSION TITLE
Title not editable in config/_default/params.toml

### DISCUSSION THREAD

**Discussion by joel-marks** [thuliteio/doks#1069] [Q&A]
Hi there. Another rookie question.
Working with the starter theme.
As I understand, editing config/_default/params.toml with changes to 

```
## Homepage
title = "My title goes here"
titleSeparator = "-"
titleAddition = "Modern Documentation Theme"
description = "Doks is a Hugo theme for building secure, fast, and SEO-ready documentation websites, which you can easily update and customize."
```

Should, I think, update the main landing page. However it is not.
Here is the layouts/index.html
I've added a css class just to change the font. This seems to have worked; there is no other css.

```{{ define "main" }}
<section class="section container-fluid mt-n3 pb-3">
  <div class="row justify-content-center">
    <div class="col-lg-12 text-center">
      <h1 class="mt-0 joel-title">{{ .Title }}</h1>
    </div>
```

Any help appreciated.
I've done an
npm run clean

**Comment by h-enk** [comment]
Your first settings should display:

![Snag_22abba37](https://github.com/h-enk/doks/assets/3902872/0606ff38-b0bb-48c0-955b-c8f8094e5025)

If you set  the title in `./content/en/_index.md`:

```md
---
title : "My Custom Title"
description: "Doks is a Hugo theme for building secure, fast, and SEO-ready documentation websites, which you can easily update and customize."
lead: "Doks is a Hugo theme for building secure, fast, and SEO-ready documentation websites, which you can easily update and customize."
date: 2020-10-06T08:47:36+00:00
lastmod: 2020-10-06T08:47:36+00:00
draft: false
images: []
---
```

It should show:

![Snag_22ae61a9](https://github.com/h-enk/doks/assets/3902872/97ee560b-a955-4566-beb0-30dd1a8f62a1)

**Reply by joel-marks** [reply]
Thank you @h-enk for your patient replies to these obvious questions. Really appreciated 😄
